Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/View.purs

```diff
@@ -110,7 +110,7 @@ view push state = let
           --   [ Anim.fadeIn state.props.enterOTP
           --   , Anim.fadeOut  (not state.props.enterOTP)
           --   ]  $
-          , enterOTPView state lang push
+          , if state.props.enterOTP then (enterOTPView state lang push) else textView[]
```

Comment:
you can remove visibility check from view if using if-else here

Reply:
Updated.